pcsx2/GS/GSCrc.cpp

```diff
@@ -17,6 +17,8 @@
 #include "GS.h"
 #include "GSCrc.h"
 
+#include "fmt/core.h"
```

Comment:
use `#include <...>` for externals/system includes

Reply:
It's a 3rdparty submodule, won't compile on linux/mac otherwise.